tcp-server.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+const net = require('net');
+
+const PORT = 8000;
+
+const server = net.createServer(socket => {
+    console.log('Client connected');
```

Comment:
I should respond with something. But at the moment I just dump the message.

Reply:
No need to respond. The client knows your connected though the magic of TCP.